docs/source-pytorch/extensions/logging.rst

```diff
@@ -231,7 +231,7 @@ Use the :func:`~pytorch_lightning.loggers.logger.rank_zero_experiment` and :func
 .. testcode::
 
     from pytorch_lightning.loggers.logger import Logger, rank_zero_experiment
-    from pytorch_lightning.utilities.distributed import rank_zero_only
+    from pytorch_lightning.utilities import rank_zero_only
```

Comment:
Note, this applies to all of these:

Deprecated is the following

`from pytorch_lightning.utilities.x import y`

because the function moved out of the file x. 

But 

`from pytorch_lightning.utilities import y`

is not, and we should keep it working for as long as possible.